aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/widget/IgnoreFieldWidget.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.aem.toolkit.test.widget;
+
+import com.exadel.aem.toolkit.api.annotations.main.Dialog;
+import com.exadel.aem.toolkit.api.annotations.main.DialogLayout;
+import com.exadel.aem.toolkit.api.annotations.widgets.DialogField;
+import com.exadel.aem.toolkit.api.annotations.widgets.IgnoreField;
+
+@Dialog(
+        name = "test-component",
+        title = "test-component-dialog",
+        layout = DialogLayout.FIXED_COLUMNS
+)
+@SuppressWarnings("unused")
+public class IgnoreFieldWidget {
+
+    @DialogField
+    @IgnoreField
```

Comment:
I think unit test for this annotation must be a little bit different. this annotation helps to manage inherited fields. E.g. we have a dialog with a couple of fields and we want another dialog that looks like the first one but without one of the fields. in this case, we can extend the second class from the first one, and use `@IngnoreField ` annotation to hide unwanted fields . So, your test case must reflect this situation. Maybe you can use one of the complex component tests, or write your own inherited dialogs.